Spanner/README.md

```diff
@@ -20,3 +20,43 @@ NOTE: In addition to the gRPC extension, we recommend installing the protobuf ex
 ```
 $ composer require google/cloud-spanner
 ```
+
+## Session warmup
+
+To issue a query against the Spanner service, the client library needs to request a session id to the server under the cover. This API call will add significant latency to your program. The Spanner client library provides a handy way to alleviate this problem by having a cached session pool.
+
+The following example shows how to use the `CacheSessionPool` with `SysVCacheItemPool` as well as how to configure a proper cache for authentication:
+
+```php
+require __DIR__ . '/vendor/autoload.php';
+
+use Google\Cloud\Spanner\SpannerClient;
+use Google\Cloud\Spanner\Session\CacheSessionPool;
+use Google\Auth\Cache\SysVCacheItemPool;
+
+$authCache = new SysVCacheItemPool();
+$sessionCache = new SysVCacheItemPool([
+    'proj' => 'B' // Use different from the default
```

Comment:
WDYT? Just helps to clarify a little bit that `proj` doesn't mean the project ID if users may not be aware.

```diff
+ Use a different project identifier for ftok than the default
- Use different from the default
```

Reply:
Done